README.md

```diff
@@ -189,20 +189,99 @@ query = Search().match(record_types=["jam"], values=["apricot"]).exclude(values=
 results = client.search(query)
 ```
 
-To filter queries provide a datetime range and a valid timezone, to override the `UTC` default. Run the following search to get all records written by a specific user during the last 24 hours:
+#### Time Filters
+
+The Search method has the capability to filter by the time records were created or last modified, restricting search results to a specific timeframe. The most basic example, of searching for all records written by a writer in the last 24 hours, is shown below:
+
 ```python
 # e3db setup...
+from e3db.types import Search
 from datetime import datetime, timedelta
+import time
+
+# Using seconds from unix epoch
+now = int(time.time())
+start = now - (60 * 60 * 24) # 60 seconds, 60 minutes, 24 hours
+end = now
 
-# time T between start <-T-> end
-end = datetime.now()
-start =  end - timedelta(days=1)
 writer_id = "some_writer_uuid"
+query = Search().match(writers=[writer_id])\
+    .range(start=start, end=end) # Filters here
+results = client.search(query)
+
+# using datetime
+now = datetime.now().astimezone()
+start = now - timedelta(days=1)
+end = now
+
+writer_id = "some_writer_uuid"
+query = Search().match(writers=[writer_id])\
+    .range(start=start, end=end) # Filters here
+results = client.search(query)
+```
+
+The Range can be open ended if you want to, for example, see all records written before yesterday:
+```python
+now = int(time.time())
+end = now - (60 * 60 * 24) # 60 seconds, 60 minutes, 24 hours == 1 day in seconds
+
+query = Search().match()\
+    .range(end=end)
+results = client.search(query)
+```
 
-query = Search().match(writers=[writer_id]).range(zone_offset="-08:00", start=start, end=end)
+Or see all records written in the last week...
+
+```python
+now = int(time.time())
+start = now - (60 * 60 * 24 * 7) # 60 seconds, 60 minutes, 24 hours, 7 days == 1 week in seconds
```

Comment:
Does Python have any constants available for time differences? Should we consider adding some helper constants in our libraries for things like `WEEK_IN_SECONDS`. I find these types of constants can _really_ improve readability when doing second based time math.

Reply:
Nothing like that as far as I'm aware, and my cursory google search didn't turn up anything either. Definitely would be helpful for users since unix epoch is the preferred range input, I'll make a note for it.